cypress/support/commands.ts

```diff
@@ -77,7 +77,7 @@ Cypress.Commands.add('signin', ({ email, password }) => {
 
 				// @TODO Remove this once the switcher is removed
 				cy.window().then((win) => {
-					win.localStorage.setItem('NodeView.migrated', 'true');
+					win.localStorage.setItem('NodeView.migrated.release', 'true');
```

Comment:
For migrating users to v2 one last time, while still allowing switching back by setting `NodeView.version` to `1` in `localStorage`